test/common/router/route_fuzz_test.cc

```diff
@@ -29,6 +29,8 @@ cleanRouteConfig(envoy::config::route::v3::RouteConfiguration route_config) {
                         envoy::config::route::v3::RouteMatch::PathSpecifierCase::
                             kHiddenEnvoyDeprecatedRegex) {
                       routes->erase(routes->begin() + i);
+                    } else if (routes->Get(i).has_filter_action()) {
```

Comment:
Can the previous removal of `hidden_envoy_deprecated_regex` be removed? It should be disallowed by default now in v3.